app/address/address-lookup.js

```diff
@@ -1,3 +1,5 @@
+const { Logger } = require('@hmcts/nodejs-logging');
```

Comment:
No need for `{ }` if using `require`. You just need `const Logger = require(...)`